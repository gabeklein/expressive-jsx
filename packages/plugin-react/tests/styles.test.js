const test = require("./_adapter");

test("basic style", `
  () => do {
    color: green;

    <this>Hello World</this>
  };
`);