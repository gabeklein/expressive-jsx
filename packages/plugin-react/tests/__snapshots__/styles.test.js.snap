// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic style 1`] = `
"() => create(\\"div\\", {
  \\"className\\": \\"do_67v\\"
});

Styles.include(\`
	.do_67v { 
		color: green;
	}
\`);"
`;

exports[`basic style 2`] = `
"() => <div className=\\"do_67v\\" />;

Styles.include(\`
	.do_67v { 
		color: green;
	}
\`);"
`;
